field_audit/migrations/0002_alter_auditevent_changed_by.py

```diff
@@ -0,0 +1,17 @@
+from django.db import migrations, models
```

Comment:
Great progress here. 

I really enjoyed going through your implementation. 😄 

A small suggestion, it would be nice if we can have this migration in the initial migration file too.

Reply:
Migrations are now squashed.